app/models/park.rb

```diff
@@ -1,4 +1,6 @@
 class Park < ApplicationRecord
+  has_many :reviews
+  
   validates :name, presence: true
   validates :description, presence: true
 end
```

Comment:
Park model tests - validations and associations

Reply:
This was added onto the photo merge.